src/test/java/org/broadinstitute/consent/http/service/MatchServiceTest.java

```diff
@@ -147,7 +147,7 @@ public void setUp() throws UnknownIdentifierException, IOException {
         when(config.getMatchURL()).thenReturn("http://ontology.org/match");
         when(dataAccessRequestDAO.findByReferenceId("NullDar")).thenReturn(null);
         when(consentDAO.findConsentById("NullConsent")).thenReturn(null);
-        when(consentDAO.findConsentById("AbsentConsent")).thenThrow(UnknownIdentifierException.class);
+        Mockito.doAnswer(invocationOnMock -> { throw new UnknownIdentifierException("AbsentConsent"); }).when(consentDAO).findConsentById("AbsentConsent");
```

Comment:
minor nit, not sure if `doAnswer` is static, but if it is, prefer to import the static.